sync/class.jetpack-sync-defaults.php

```diff
@@ -104,6 +104,8 @@ class Jetpack_Sync_Defaults {
 		'users_can_register',
 		'active_plugins',
 		'uninstall_plugins',
+		'advanced_seo_front_page_description',
+		'advanced_seo_title_formats',
 	);
```

Comment:
Added in aa7ed7adc93798679fb9dee347853233401633dd. Although it might be better to use the existing `Advanced_SEO_Posts::DESCRIPTION_META_KEY` constant, I've decided to leave `advanced_seo_description` string there in order to keep it consistent with other options in that list.